lxd/device/unix_hotplug.go

```diff
@@ -0,0 +1,212 @@
+package device
```

Comment:
This file is causing `go get` to fail on both macOS and Windows. That's because `go-libudev` cannot be imported on those platforms. To fix it, adding a build tag to this file should make `go get` behave.

```suggestion
// +build linux,cgo

package device
```